unitTests/utils/DomTest.ts

```diff
@@ -32,6 +31,39 @@ export function DomTests() {
         Simulate.removeJQuery();
       });
 
+      describe('without custom event (IE11)', () => {
+        let customEvent;
+        beforeEach(() => {
```

Comment:
Atm, with `beforeEach` and `afterEach`, you remove the `CustomEvent` property from the window object, run the test, then readd it? then remove it?

Maybe they should be `beforeAll` and `afterAll` if you want to execute the tests in an environment that does not have `CustomEvent` on the window?